certbot-ci/certbot_integration_tests/utils/misc.py

```diff
@@ -277,12 +277,22 @@ def read_certificate(cert_path: str) -> str:
     :param str cert_path: the path to the certificate
     :returns: the TEXT version of the certificate, as it would be displayed by openssl binary
     """
+    cert = get_certificate(cert_path)
+    return crypto.dump_certificate(crypto.FILETYPE_TEXT, cert).decode('utf-8')
+
+def get_certificate(cert_path: str) -> crypto.X509:
+    """
+    Load the certificate from the provided path, and return a python representation
+    of it.
+    :param str cert_path: the path to the certificate
+    :returns: the X509 version of the certificate
```

Comment:
Just some minor nits about this docstring.

```suggestion
    Load the certificate from the provided path.
    :param str cert_path: the path to the certificate in PEM format
    :returns: the certificate
```